app/views/admin/auctions/show.html.erb

```diff
@@ -19,13 +19,13 @@
 <p><%=h raw @auction.html_description %></p>
 
 <div class='usa-grid'>
-  <% if @auction.current_bid? %>
+  <% if @auction.highlighted_bid? %>
```

Comment:
I think this would be a future refactoring indeed. I just wanted to rename `current_bid` to something a bit more descriptive